packages/drawer/README.md

```diff
@@ -76,6 +79,69 @@ class MyApp extends Component {
 }
 ```
 
+#### Permanent Drawer Below Top App Bar Variant
```

Comment:
Can we cut this section and make the `Usage with Top App Bar` section more general so that it applies to both the permanent and dismissible drawer?

Reply:
Good suggestion..pushed changes. Also fixed a slight bug with screenshot tests